cloudwatchmon/cli/put_instance_stats.py

```diff
@@ -302,11 +303,19 @@ def config_parser():
                             type=to_lower,
                             choices=size_units,
                             help='Specifies units for disk space metrics.')
-     disk_group.add_argument('--disk-inode-util',
+    disk_group.add_argument('--disk-inode-util',
                             action='store_true',
                             help='Reports disk inode utilization in percentages.')
 
 
+    process_group = parser.add_argument_group('process metrics')
+    process_group.add_argument('--process-name',
```

Comment:
Please use action='append' (as in --disk-path) instead of comma-separated (for consistency).